internal/storage/postgres/sensor/entities/mqtt_payload.go

```diff
@@ -7,9 +7,11 @@ type Watermark struct {
 }
 
 type MqttPayload struct {
-	DeviceID    string      `json:"device_id"`
+	Device      string      `json:"device"`
 	Battery     float64     `json:"battery"`
 	Humidity    float64     `json:"humidity"`
 	Temperature float64     `json:"temperature"`
+	Latitude    float64     `json:"latitude"`
+	Longitude   float64     `json:"longitude"`
```

Comment:
Why are latitude and longitude being added here? These values are already saved once directly as sensor attributes, so there’s no need to include them again in the sensor data, every time the data is stored.

We’ve previously discussed the structure of sensor data in #233.